tools/deployment/pytorch2onnx.py

```diff
@@ -30,8 +30,13 @@ def pytorch2onnx(model,
         'input_path': input_img,
         'normalize_cfg': normalize_cfg
     }
+
+    if torch.cuda.is_available():
+        model.cuda()
+    device = next(model.parameters()).device
+
     # prepare input
-    one_img, one_meta = preprocess_example_input(input_config)
+    one_img, one_meta = preprocess_example_input(input_config, device)
```

Comment:
`device` argument is missed in  Line 149.

Reply:
Fixed.